File: psup assignment/cpp/q1.cpp
#include <iostream>

void positiveCheck() {
    int num;
    std::cout << "Enter a number: ";
    std::cin >> num;
    if (num > 0) {
        std::cout << "The number is positive." << std::endl;
    }
}

int main() {
    positiveCheck();
    return 0;
}

File: psup assignment/nodejs/q1.js
const readline = require('readline');
function positiveCheck() {
    const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
    rl.question('Enter a number: ', (num) => {
        if (parseInt(num) > 0) {
            console.log('The number is positive.');
        }
        rl.close();
    });
}